lib/scummvm/config.py

```diff
@@ -0,0 +1,67 @@
+DEBUGGER = True
+LOG_DIRECTORY = "/home/rohan/parser_files/ScummVM/"
+OUTPUT_DIRECTORY = "/home/rohan/Desktop/temp/"
+STARTING_DATE = "2016-1-1"
+ENDING_DATE = "2016-1-31"
+MINIMUM_NICK_LENGTH = 3
+
+# plotly login credentials
+USER_NAME = 'rohangoel963'
+TOKEN = 'WUqptWKXPareXfFMBtDG'
+
+# change to ["ALL"] if want to perform analysis over all channels
+CHANNEL_NAME = ["#scummvm"]
```

Comment:
In slack-gc.py, I left this to #kubuntu-devel. How does the channel name effect the analysis. If the name has any side effect on the slackware analysis, then it is better to change [line-13 of lib/slack/config.py](https://github.com/prasadtalasila/IRCLogParser/blob/dev/lib/slack/config.py#L13).

Reply:
`slack-gc.py` doesn't use channel_name from config file because there is only one 'channel'=slackware so changing/keeping that variable will have no affect - hence it might be a good idea to remove it all together from that config?

Agreed that wrt to scummvm we also have the similar situation, and I can do away with the variable here too, but I intentionally didn't do that because that'd mean creating a new reader for scummvm logs. Note: Presently for scummvm, there is only nick tracker which is different. All the other files are same as ubuntu and used from lib/analysis and not lib/scummvm/analysis.